pkg/apis/dynatrace/v1alpha1/dynakube_types.go

```diff
@@ -0,0 +1,127 @@
+package v1alpha1
+
+import (
+	"github.com/operator-framework/operator-sdk/pkg/status"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DynaKubeSpec defines the desired state of DynaKube
+// +k8s:openapi-gen=true
+type DynaKubeSpec struct {
+	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+	BaseDynaKubeSpec `json:""`
+
+	OneAgentAPMSpec OneAgentAPMSpec `json:"oneAgentApm,omitempty"`
+
+	OneAgent OneAgentSpec `json:"oneAgent,omitempty"`
```

Comment:
We don't need these, OneAgent features won't be supported on the DynaKube CRD anytime soon (or ever.) Remove the types as well.